docker/Dockerfile

```diff
@@ -1,63 +1,57 @@
-FROM condaforge/mambaforge:23.3.1-1
+FROM mambaorg/micromamba:1.5.1
+
+USER root
 
 # Install curl and js
 # awscli requires unzip, less, groff and mandoc
 # hadolint ignore=DL3008
-RUN apt-get update && apt-get install --no-install-recommends -y curl jq unzip less groff mandoc \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y git curl jq unzip less groff mandoc && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
 # Configure gsutil authentication
 # hadolint ignore=DL3059
 RUN printf '[GoogleCompute]\nservice_account = default' > /etc/boto.cfg
 
 # Install awscli2
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install
-
-# Create a non-root user inside the container
-# hadolint ignore=DL3059
-RUN useradd -Ums /bin/bash catalyst
-
-ENV CONTAINER_HOME=/home/catalyst
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
```

Comment:
I guess this is what Amazon wants us to do?!?

Reply:
And who are we to question Lord Bezos?